plugins/org.eclipse.n4js/src/org/eclipse/n4js/N4JSGlobals.java

```diff
@@ -95,6 +95,16 @@
 	 */
 	public static final String N4JSD_FILE_EXTENSION = "n4jsd";
 
+	/**
+	 * Files extension of TS files (<b>not</b> including the separator dot).
+	 */
+	public static final String TS_FILE_EXTENSION = "ts";
+
+	/**
+	 * Files extension of TS definition files (<b>not</b> including the separator dot).
+	 */
+	public static final String DTS_FILE_EXTENSION = "d.ts";
```

Comment:
File extension `.d.ts` has to be added also to the watcher glob in the LSP extension, to get file change events in the incremental builder. Here:
https://github.com/eclipse/n4js/blob/4791200c5f5bc051497cb0047121cdaa6aad665c/lsp-clients/n4js-vscode-extension/src/ExtensionProvider.n4js#L95